### ISSUE TITLE
Missing URL tables

### ISSUE THREAD

**Issue by Ismael-Castillo** [FBoucher/AzUrlShortener#441]
Once deployed and inspecting in Azure Storage Explorer, only able to see $Metrics tables but no URL tables.  Did I miss a step?

**Comment by Ismael-Castillo** [comment]
I was able to find answer in discussions post.  For anyone with same questions, tables are created if they don't yet exist.

**Comment by FBoucher** [comment]
This is correct. Tables are created at the first call when they don't exist.
[ ] Update docs to specify this. I Good place could be [here](https://github.com/FBoucher/AzUrlShortener/wiki/How-to-deploy-your-AzUrlShortener#validate-the-deployment)

**Comment by FBoucher** [comment]
Updated

**Comment by FBoucher** [comment]
Once the deployment of AzUrlShortener is finished, [Validate the deployment](https://github.com/FBoucher/AzUrlShortener/wiki/How-to-deploy-your-AzUrlShortener#validate-the-deployment)
This will create the table **UrlsDetails**. It will be empty but it will be present.

Then you can create your first Short URL by following the instruction here [Create a new Short URL](https://github.com/FBoucher/AzUrlShortener/wiki/How-to-Use-Azure-Storage-Explorer-as-Admin-Tools-for-AzUrlShortener#create-a-new-short-url)

> The table with the clicks will be automatically created when a link will be used.

**Comment by Ismael-Castillo** [comment]
Thank you. Great project.
